_posts/2023-04-14-torch-0-10/torch-0-10.Rmd

```diff
@@ -0,0 +1,176 @@
+---
+title: "torch 0.10.0"
+description: >
+  torch v0.10.0 is now on CRAN. This version upgraded the underlying LibTorch to 1.13.1,
+  added support for Automatic Mixed Precision. As an experimental feature we now
+  also support pre-built binaries, so you can install torch without having to deal
+  with the CUDA installation.
+author:
+  - name: Daniel Falbel
+    affiliation: Posit
+    affiliation_url: https://www.posit.co/
+slug: torch-0-9-0
+date: 2023-04-21
+categories:
+  - Torch
+  - Packages/Releases
+  - R
+output:
+  distill::distill_article:
+    self_contained: false
+    toc: true
+preview: images/torch.png
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = TRUE, eval = FALSE, fig.width = 6, fig.height = 6)
+```
+
+torch v0.10.0 is now on CRAN. This version upgraded the underlying LibTorch to 1.13.1,
```

Comment:
Hm, this will duplicate the abstract, and thus appear twice on the web page ... We'd need to completely remove the paragraph, and have some other short intro instead. How about:

Today, we're happy to announce the first double-digit release of torch. It brings significant enhancements in both functionality and usability.

(((I'm aware it's double-digit in the minor version ... you could add a footnote, saying "in the minor version, but still :-)" ... or something like that)))

Reply:
Thanks! I didn't notice that this appears also in the top page. I have updated the intro.